tap_quickbooks/quickbooks/rest.py

```diff
@@ -103,6 +103,13 @@ def _sync_records(self, url, headers, params, stream):
         max = 100
         while True:
             headers.update(self.qb._get_standard_headers())
+            records_deleted = []
+            if self.qb.include_deleted:
+                # Get the deleted records first
+                params['query'] = f"{query} where Active = false  STARTPOSITION {offset} MAXRESULTS {max}" 
```

Comment:
This logic doesn't seem right, did you test it locally @AlvaroRaul7 ?

Reply:
yes, I tested it locally also, there is a [https://help.developer.intuit.com/s/question/0D54R00007y3qZrSAI/how-to-get-inactive-vendors-or-accounts-from-quickbooks-looking-for-query](thread) that explains the query